src/cart/internal-line-item.ts

```diff
@@ -12,6 +12,7 @@ export default interface InternalLineItem {
     name?: string;
     quantity: number;
     brand?: string;
+    categoryNames?: Array<{ key: string | number, value: string }>;
```

Comment:
I don't quite follow along with why this is an array of objects with key and value properties. Can you elaborate? I thought the API output looked like: `['category 1/child 1', 'category 2/child 2']`.